README.md

```diff
@@ -32,7 +27,6 @@ $ yarn global add deyarn
 
 ## Usage
 
-(in the directory of the project to convert, which must be a Git repository)
```

Comment:
This still does have to be done in the directory of the project to convert right? Though it no longer needs to be a Git repository.

Reply:
Done